tests/core/test_dataobject.py

```diff
@@ -196,3 +196,15 @@ def test_user_dict_persists_with_pack_labels_filter():
     image.user_dict['name'] = 'image'
     image = image.pack_labels()
     assert image.user_dict['name'] == 'image'
+
+
+def test_user_dict_persists_with_point_voxels_to_cell_voxels(uniform):
```

Comment:
There are still a few references to the old "point voxels to cell voxels" names / terminology which I will update.
```suggestion
def test_user_dict_persists_with_points_to_cells(uniform):
```